nautobot/docs/user-guide/administration/installation/health-checks.md

```diff
@@ -0,0 +1,307 @@
+# Nautobot Health Checks
+
+In a production deployment of Nautobot, you'll want health checks (also termed liveness checks, readiness checks, etc.) for each distinct component of the Nautobot system, to be able to detect if any component fails and ideally respond automatically. While this topic can be (and is) the subject of multiple books, this document attempts to provide some basic "best practices" guidelines. If you're deploying Nautobot as part of a larger enterprise system, of course you'll want to follow your organization's experts and their guidance, but if you're "on your own", you can do worse than starting here.
+
+## Health Check Approaches
+
+In general the following commands or HTTP requests can serve as health checks for the various components of a Nautobot system.
+
+### Nautobot HTTP Server
+
+In addition to simply monitoring the existence of the `nautobot-server` process ID, two more in-depth approaches are possible here.
+
+An HTTP `GET` request to `<server>/health/` should return an HTTP `200 OK` response so long as:
+
+- the server is running
+- and the server can connect to the database
+- and all Django migrations have been applied
```

Comment:
Crossing my mental wires here - the migrations check is only in 2.2/`next` (#5342) but this PR is targeted at `develop`. I'll add a note to that effect.